R/palaeorotate.R

```diff
@@ -22,12 +22,14 @@
 #' for fossil occurrences. Either "grid" (default) to use reconstruction files,
 #' or "point" to use the GPlates API service. See details section for specific
 #' details.
-#' @param uncertainty \code{logical}. Should the uncertainty in
-#' palaeogeographic reconstructions be returned? If set to TRUE, the
-#' palaeocoordinates from all reconstruction files (models) are returned, along
-#' with their respective palaeolatitudinal range and the maximum Great
-#' Circle Distance between palaeocoordinates (in km). This argument is only
-#' relevant if `method` is set to "grid".
+#' @param uncertainty \code{NULL} or \code{character}. The names of the plate
+#' rotation models to calculate palaeogeographic uncertainty for.
+#' Palaeocoordinates from all reconstruction files (models) will be returned,
+#' along with their respective palaeolatitudinal range and the maximum Great
+#' Circle Distance between palaeocoordinates (in km).
+#' By default, this argument is set to NULL (default) and palaeogographic
+#' uncertainty is not calculated. This argument is only relevant if `method`
+#' is set to "grid".
```

Comment:
I have implemented point 1 and 3. For point 2, I think `uncertainty` should remain as `FALSE` by default. The reason for this is that I imagine most people will only want to use one model at a time in the palaeorotate function. It would be tiresome to continuously set the argument to `FALSE`. Of course we could ignore this argument when only one model is provided, but I think this is unnecessary.

Reply:
My thought for #2 was indeed that we could just ignore the argument if only one model is specified.

Reply:
Implemented!